cmd/add/add_test.go

```diff
@@ -1 +1,118 @@
 package add
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func TestAddCmd(t *testing.T) {
+	testFileName := "add-me-please"
+	type addOptionsTestCase struct {
+		name         string
+		args         []string
+		fileContents string
+		wantErr      bool
+		wantAdded    string
+	}
+	var mockAddCmd = func(tc addOptionsTestCase) *cobra.Command {
+		cmd := &cobra.Command{
```

Comment:
This `cmd` variable is the same as the exported Cmd from add.go. Is it possible to just use that command for my tests? Would that create problems?